ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/TransactionCommitterSpec.scala

```diff
@@ -289,6 +293,78 @@ class TransactionCommitterSpec
     }
   }
 
+  "out of time bounds entry" should {
+
+    "be set" when {
```

Comment:
This should probably be "always", which means we should test it for each validation outcome. Is that right? If so, I think we can map it as a coverage improvement (i.e., not necessarily as part of this PR, also considering that the `main` one is already merged and this fix is prioritary).

Reply:
Indeed, it should be always set for each validation outcome. Will add a ticket to improve testing coverage there